src/common/config.cc

```diff
@@ -946,83 +952,177 @@ int md_config_t::set_val_impl(const char *val, const config_option *opt)
 
 int md_config_t::set_val_raw(const char *val, const config_option *opt)
 {
+  const int CEPH_SETTING_SAFE = 1;
```

Comment:
@oliveiradan Why local variable is taken as caps?
Conventionally we take macros as this?